### DISCUSSION TITLE
Using TCXO with SX1262

### DISCUSSION THREAD

**Discussion by diocorreia** [Lora-net/LoRaMac-node#1044] [Q&A] [answered]
Hi,

I have tried your lib with the SX1261MB2BAS. After calling SX126xSetDio3AsTcxoCtrl, DIO3 is set to the right value. However, when trying the same sample code with a custom pcb (with SX1262 chip), DIO3 is set to LOW instead. Any idea of what is happening?

Kind regards,
Diogo Correia

**Comment by mluis1** [comment, marked as answer]
Unfortunately, there is not enough information about the issue you are encountering. So, no idea on what is happening.

Would it be possible to provide your `sx1262-custom-board.c` file?

Please also note that `SX1261MB2BAS` shield has no TCXO. Did you meant `SX1262MBXDAS`?

In the future it would be nice if you could post this kind of questions on the project [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab. It is a better place to engage discussions and then we can agree if it is an issue or not.
If you don't mind I move this issue to the [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab.

**Reply by diocorreia** [reply]
My apologies for the slow reply; I already solved the issue, the problem was that my CS pin (on the custom board file) was set with inverted logic (high at 0).

Kind regards,
Diogo Correia
